js/ooyala_ssai.js

```diff
@@ -815,7 +815,10 @@ OO.Ads.manager(() => {
       const mainUrlParts = mainUrl.split('/');
       if (mainUrlParts !== null) {
       	this.domainName = mainUrlParts[2];
-      	this.currentEmbed = mainUrlParts[4];
+        if(mainUrlParts[3] === 'hls')
```

Comment:
If you are using this `if` statement to identify VOD from Live assets I think you should use `amc.isLiveStream` in order to get the asset type.

Reply:
Addressed it.